Workaround some issues with the pulumi build pipeline.
Currently the build system doesn't quite provide a way to build
just the native components we need to apk package.

diff --git a/Makefile b/Makefile
index 0b81a9de2..26a2f8e8a 100644
--- a/Makefile
+++ b/Makefile
@@ -53,6 +56,8 @@ install:: generate

 install_all:: install

+install_plugin_all:: install
+
 dist:: build
 	cd pkg && go install -ldflags "-X github.com/pulumi/pulumi/pkg/v2/version.Version=${VERSION}" ${PROJECT}

diff --git a/build/common.mk b/build/common.mk
index 20690ac23..c49c5fbd3 100644
--- a/build/common.mk
+++ b/build/common.mk
@@ -128,6 +128,7 @@ only_test_fast:: $(SUB_PROJECTS:%=%_only_test_fast)
 default:: $(SUB_PROJECTS:%=%_default)
 all:: $(SUB_PROJECTS:%=%_all)
 install_all:: $(SUB_PROJECTS:%=%_install_all)
+install_plugin_all:: $(SUB_PROJECTS:%=%_install_plugin_all)
 test_all:: $(SUB_PROJECTS:%=%_test_all)
 ensure:: $(SUB_PROJECTS:%=%_ensure)
 dist:: $(SUB_PROJECTS:%=%_dist)
@@ -212,6 +213,8 @@ $(SUB_PROJECTS:%=%_build):
 	@$(MAKE) -C ./$(@:%_build=%) build
 $(SUB_PROJECTS:%=%_install_all):
 	@$(MAKE) -C ./$(@:%_install_all=%) install
+$(SUB_PROJECTS:%=%_install_plugin_all):
+	@$(MAKE) -C ./$(@:%_install_plugin_all=%) install_plugin
 $(SUB_PROJECTS:%=%_lint):
 	@$(MAKE) -C ./$(@:%_lint=%) lint
 $(SUB_PROJECTS:%=%_test_fast):
diff --git a/sdk/nodejs/Makefile b/sdk/nodejs/Makefile
index 3c5769223..04d92bcfb 100644
--- a/sdk/nodejs/Makefile
+++ b/sdk/nodejs/Makefile
@@ -32,11 +32,11 @@ build_plugin::

 build:: build_package build_plugin

-install_package:: build
+install_package:: build_package
 	cp dist/pulumi-resource-pulumi-nodejs "$(PULUMI_BIN)"
 	cp dist/pulumi-analyzer-policy "$(PULUMI_BIN)"

-install_plugin:: build
+install_plugin:: build_plugin
 	GOBIN=$(PULUMI_BIN) go install -ldflags "-X github.com/pulumi/pulumi/sdk/v2/go/common/version.Version=${VERSION}" ${LANGUAGE_HOST}

 install:: install_package install_plugin
